src/components/ProductTours/AdminOnboardingTours/AdminOnboardingTours.tsx

```diff
@@ -98,7 +104,7 @@ const AdminOnboardingTours: FC<AdminOnboardingToursProps> = ({
 
   return (
     <>
-      <CheckpointOverlay target={targetSelector} />
+      <CheckpointOverlay target={learnerProgressSteps[currentStep]?.target || targetSelector} />
```

Comment:
Why do we need this OR? And what about for the other flows other than learnerProgress?